### PR TITLE
feat: add fast-components package

### PR DESCRIPTION
**Pull request by nicholasrice**
# Description
Adds a package that will house Web Component implementations. Configures a Storybook development environment for validating experiences and configurations of components.

Also adds @EisenbergEffect's beautiful `name-tag` component as an example to illustrate how the pieces should come together to form the exports from a component directory.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by eljefe223** [comment]
smoked.

File: packages/fast-components/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# FAST Components
+
+## Development
+
+To start the component development environment, run `yarn start`.
+
+### Known issue with Storybook site hot-reloading
+
+Storybook will watch modules for changes and hot-reload the module when necessary. This is usually great, however it poses a specific problem when the module being hot-reloaded defines a custom element. A custom element name can only be defined by the `CustomElementsRegistry` once, so reloading a module that defines a custom element will attempt to re-register the custom element name, throwing an error because the name has already been defined. This error will manifest with the following message:
```
Comment (chrisdholt) [inline packages/fast-components/README.md]: super nit:

```
This is usually great; however, it poses a specific problem...
```

File: packages/fast-components/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@storybook/html": "^5.3.13",
+    "prettier": "1.14.3",
+    "typescript": "^3.7.5"
+  },
+  "dependencies": {
+    "@microsoft/fast-element": "^0.0.0",
+    "@storybook/cli": "^5.3.13",
+    "awesome-typescript-loader": "^5.2.1"
```
Comment (janechu) [inline packages/fast-components/package.json]: I think storybook cli and awesome typescript loader go in devDependencies

Comment (nicholasrice) [reply packages/fast-components/package.json]: yep - good catch

**Pull request merged**